ConsoleApp1/TestProject1/UnitTest1.cs

```diff
@@ -0,0 +1,32 @@
+using LinqLesson;
+using Microsoft.VisualStudio.TestPlatform.TestHost;
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace TestProject1
+{
+    public class LinqUnitTest1
+    {
+
+ public LinqLesson.ProgramX _program = new LinqLesson.ProgramX();
+
+        [Fact]
+        public void MaxCompanyTest()
+        {
+
+            //ARRANGE
+            List<Person> people = new List<Person>{
```

Comment:
not critical, but in future, you do not need to put everything person-related in arrange, you need only data that can change the result